kernel/Cargo.toml

```diff
@@ -159,3 +159,4 @@ tracing-subscriber = { version = "0.3", default-features = false, features = [
   "env-filter",
   "fmt",
 ] }
+async-trait = "0.1" # only used for our custom SlowGetStore ObjectStore implementation
```

Comment:
nit: keep alphabetical.

Reply:
moved up, to after our path-based deps but on top of others, though they aren't in order it doesn't look like..

Reply:
hah right, we should actually alphabatize those at some point :)